dev/linter.dockerfile

```diff
@@ -12,7 +12,9 @@ RUN apk add --no-cache \
   python3~=3.9 \
   python3-dev~=3.9 \
   musl-dev~=1.2 \
-  py3-pip~=20.3
+  py3-pip~=20.3 \
+  nodejs~=16.20 \
+  npm~=8.1.3
```

Comment:
Any reason why this was selected over other versions?

Reply:
I figured out those versions by building the image without specifying a version and then running the container and inspecting what versions were installed :joy: 

I am interested to learn if you have an approach or process for determining which version?

Reply:
I was actually curious why older versions were used over the latest. It makes sense to use the versions that work/come with the default alpine image.

I wonder if it will also make sense to add a command to update the apt package repositories and download the latest node/npm versions. That said I'm not particularly worried if the linter container breaks, since it's non-essential, so this is by no means pressing. Thoughts?